src/utils/Math.vy

```diff
@@ -0,0 +1,96 @@
+# @version ^0.3.7
+
+
+@external
+@view
+def wad_cbrt(x: uint256) -> uint256:
+    """
+    @notice Calculate the cubic root of a number in 1e18 precision
+    @dev Consumes around 1500 gas units
+    @param x The number to calculate the cubic root of
+    @return The cubic root of the number
+    """
+
+    # Since this cubic root is for numbers at 1e18 base, we need to scale the
+    # input by 1e36 to increase precision. This will overflow for very large
+    # numbers. So conditionally sacrifice precision.
+
+    xx: uint256 = 0
+    if x >= 115792089237316195423570985008687907853269 * 10**18:
```

Comment:
actually it is just `max_value(uint256) / 10**36`, which has no additional costs than using the number itself.

Reply:
Actually it's not related to gas savings why I want to use a constant here, but rather composability. Maybe you need that constant for another function at one point and you can simply use it by calling the constant, or once lib modules are supported, you can import the constant if needed.

Reply:
True, but I doubt anyone needs to import this since it is available for free from the compiler everywhere: so it isn't something that the library offers that the compiler doesn't.

I will add it for constants like ln(2), though!